standup_notes/__main__.py

```diff
@@ -194,16 +194,17 @@ def delete_notes(date_to_delete):
     date_in_int = int(date_to_delete.replace('-', ''))
     for file in os.listdir(STANDUP_NOTES):
         value = int(file.split('.')[0])
```

Comment:
If I remember correctly, the problem starts with this line for .swp files since they start with a `.`. `value` is None and can't be compared to `date_in_int` so an exception is thrown. So you need to check and make sure the file ends with `.txt` **before** comparing value with the date.